tests/lib/prepare.sh

```diff
@@ -3,7 +3,8 @@
 set -eux
 
 prepare_classic() {
-    apt install -y ${SPREAD_PATH}/../snapd_*.deb
+    # relying on dpkg as apt(-get) does not support installation from local files in trusty.
+    dpkg -i ${SPREAD_PATH}/../snapd_*.deb || apt-get -f install -y
```

Comment:
Yup, I think this change would benefit from an apt helper function taking care of conditionally executing the right cmd then. What do you think?